typeconv.go

```diff
@@ -29,6 +29,14 @@ func Load(conf loader.Config, args []string) (*loader.Program, []types.Error, er
 	if _, err := conf.FromArgs(args, true); err != nil {
 		return nil, nil, err
 	}
+	conf.TypeCheckFuncBodies = func(path string) bool {
+		// TODO(haya14busa): skip check in func bodies for dependent packages.
+		_, ok := stdlibs[path]
+		if ok {
```

Comment:
**[gosimple]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should use 'return <expr>' instead of 'if <expr> { return <bool> }; return <bool>' (S1008)